docs/administering/OFFBOARDING_A_FUND.md

```diff
@@ -0,0 +1,165 @@
+  models_fund_dependent.each do |model|
+    fund_counts[model.to_s.to_sym] = model.where(fund_id: fund.id).size
+  end
+  post_deletion_counts[fund.name] = fund_counts
+end
+counts = {}
+models_fund_agnostic.each do |model|
+  counts[model.to_s.to_sym] = model.all.size
+end
+post_deletion_counts['fund_agnostic'] = counts
+
+#save final counts to file
+$stdout = File.new('post_deletion.out', 'w')
+$stdout.sync = true
+post_deletion_counts
+$stdout = STDOUT
+
+exit
+```
+
+diff those files!
+
+  - The other fund objects should not have changed significantly
+  - There should be a jump in PaperTrails corresponding to the total object count we've deleted
+
+`diff -y initial_count.out post_deletion.out`
+
+IF AND ONLY IF everything looks good, let's continue.
+
+Remove the PaperTrails associated with the fund, and finally the Fund row itself.
+
+run `rails c` again
+
+```ruby
+# delete the PaperTrails
+#  🚨 ⚠️ 🚨 ⚠️ 🚨 This is the point of no return.
+# Once these are deleted, we cannot restore the funds information from this database
+Fund.all.pluck(:name,:id)
+defunct_fund_id = #the fund you are removing
+defunct_fund_versions = PaperTrail::Version.where_object(fund_id: defunct_fund_id)
+defunct_fund_versions.destroy_all
+defunct_fund_changes = PaperTrailVersion.where_object_changes(fund_id: defunct_fund_id)
+defunct_fund_changes.destroy_all
+
+# Finally, destroy the fund row
+fund = Fund.find(defunct_fund_id)
+fund.destroy
+
+exit
+```
+
+Copy any exerpts from the output files you want for later reference and `exit` the bash
+
+### Removing the Subdomain Configuration
+
+#### Remove Google OAuth
+  - Go to our [GCP project](https://console.cloud.google.com/apis/credentials?project=daria-services-multitenant)
+  - Open up DARIA Services Production from the list of Client IDs to edit
+  - Remove the Authorized redirect URI for the fund's subdomain
+#### Remove the DNS in Namecheap
+  - Open up the Namecheap control panel and head to Advanced DNS.
+  - Remove the CNAME Record for the host
+#### Remove the subdomain in heroku
+  - Open up the pipeline in Heroku and go to the production instance, then open up Settings, then go to Domains.
+  - Click the edit icon for the fund's domain and select Remove Domain
+  - Remove the domain's certs, if not done automatically
+#### Remove defunct manual database backups in heroku
+  - Open up the pipeline in Heroku and go to the production instance, then open up Resources, then click on Heroku Postgres, then open Durability
+  - Remove all Manual Backups & Data Exports
+
+### Remaining Fund Information
+
+#### Sentry
+Sentry errors for the fund, containing no PII, may be retained for up to 90 days.
+
+#### Database backups
+Automatic database backups are retained for four days.
+
+In the event that we need to rollback the database we will repeat the removal of all fund information.
+
+### Removing the fund from our contact systems
+
+  - Remove the fund from our Fund Information & Point of Contact document
```

Comment:
I think we can leave em on this and just mark the row defunct -- I can think of good reasons that we'd want to continue to be in touch that would outweigh risk imo